templates/pages/example.html.twig

```diff
@@ -57,10 +58,14 @@
         <p>{{ content.single_account_selection.name }}</p>
     {% endif %}
 
+    <p>{{ content.account_selection | map(account => account.name) | join(', ') }}</p>
```

Comment:
As we follow the symfony code style, there should be no spaces between the filters `|` /cc @nnatter